qa/L0_batcher/batcher_test.py

```diff
@@ -56,14 +56,10 @@
     USE_GRPC = False
 assert USE_GRPC or USE_HTTP, "USE_GRPC or USE_HTTP must be non-zero"
 
-BACKENDS = os.environ.get('BACKENDS',
-                          "graphdef savedmodel onnx libtorch plan custom")
+BACKENDS = os.environ.get('BACKENDS', "graphdef savedmodel onnx libtorch plan")
 
 _trials = BACKENDS.split(" ")
-if "custom" in BACKENDS:
-    _ragged_batch_supported_trials = ("custom",)
-else:
-    _ragged_batch_supported_trials = ()
+_ragged_batch_supported_trials = ()
```

Comment:
@GuanLuo I think we should enable ragged batch testing for the other backends in a different ticket since there is already way to much going on here